diff -up libburn-0.7.0/libburn/write.c.noclose libburn-0.7.0/libburn/write.c
--- libburn-0.7.0/libburn/write.c.noclose	2012-05-21 19:45:58.697670711 +0200
+++ libburn-0.7.0/libburn/write.c	2012-05-21 19:47:56.074719406 +0200
@@ -868,8 +868,19 @@ ex:;
 		d->sync_cache(d);
 
 		/* ts A61030 */
-		if (burn_write_close_track(o, s, tnum) <= 0)
-			ret = 0;
+		/* ts A91003 :
+		   At least in simulation mode this causes NWA=0 for the
+		   next track. cdrecord does not use CLOSE TRACK at all but
+		   ends the tracks by SYNCHRONIZE CACHE alone.
+		*/
+		/* ts A91202 :
+		   Peng Shao reports that his LG GH22LS30 issues an SCSI error
+		   on CLOSE TRACK even in non-dummy mode. So i better give up
+		   this gesture which seems not be needed by any drive.
+			if (!o->simulate)
+				if (burn_write_close_track(o, s, tnum) <= 0)
+					ret = 0;
+		*/
 	}
 	return ret;
 }
